### PR TITLE
test: migrage trace e2e to ginkgo implement

### PR DESCRIPTION
**Pull request by zaunist**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [ ] New feature provided
- [x] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

Please update this section with detailed description.

**Related issues**

part of #2197 

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [x] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by zaunist** [comment]
cc @bzp2010 @nic-chen @starsz @okaybase @juzhiyuan

File: .github/workflows/backend-e2e-test.yml [hunk 1]
```diff
@@ -92,40 +92,12 @@ jobs:
         working-directory: ./api/test/e2e
         run: go test -v
 
-      - name: re-generate json schema
-        working-directory: ./api/
-        run: |
-          curl 127.0.0.1:9090/v1/schema > ./conf/schema.json
```
Comment (starsz) [inline .github/workflows/backend-e2e-test.yml:98]: So it's useless code ?

Comment (zaunist) [reply .github/workflows/backend-e2e-test.yml:98]: Yes, we don't need this step anymore

**Review by nic-chen — APPROVED** [review]

**Comment by zaunist** [comment]
cc @bzp2010

**Review by okaybase — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**